backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/ApiUserManagementTest.java

```diff
@@ -897,6 +898,81 @@ void getUsers_orgUser_failure() {
     runQuery("users-query", "Current user does not have permission to request [/users]");
   }
 
+  @Test
+  void setCurrentUserTenantDataAccess_adminUser_success() {
+    useSuperUser();
+    ObjectNode variables =
+        JsonNodeFactory.instance
+            .objectNode()
+            .put("organizationExternalId", TestUserIdentities.DEFAULT_ORGANIZATION)
+            .put("justification", "This is my justification");
+    ObjectNode user =
+        (ObjectNode)
+            runQuery(
+                    "set-current-user-tenant-data-access",
+                    "SetCurrentUserTenantDataAccessOp",
+                    variables,
+                    null)
+                .get("setCurrentUserTenantDataAccess");
+    assertEquals("ruby@example.com", user.get("email").asText());
+    assertEquals(Role.ADMIN, Role.valueOf(user.get("role").asText()));
+    Set<UserPermission> allPermissions =
```

Comment:
i think there's a built-in syntactic sugar function for getting all the values of an enum class. i don't think you need to literally enumerate them all here.